src/handlers.js

```diff
@@ -1,4 +1,47 @@
 // 1. handler to return the homepage('index.html and all resources from
-      //index.html')
+//index.html')
 // 2. handles search --> making a call to request.js and processes the result;
 // 3. handles other and returns 404 error.
+const path = require("path");
+const fs = require('fs');
+const apiRequest = require('./apiRequest');
+const router = require('./router.js');
+
+let handlers = module.exports = {}
```

Comment:
not sure what's going on here; we do need to export handlers at the bottom so that router can reference it.